Assets/Scripts/Character/ItemHolder.cs

```diff
@@ -5,31 +5,46 @@
 public class ItemHolder : MonoBehaviour {
   [SerializeField] private Transform handTransform;
   [SerializeField] private Item _heldItem;
+  [SerializeField] private Collider holderCollider; 
 
   public Item HeldItem { get => _heldItem; }
 
   private Transform oldParent = null;
 
+  private void ignoreCollisions(Item item, bool ignore) {
+    foreach (Collider collider in item.GetComponentsInChildren<Collider>()) {
+      Physics.IgnoreCollision(holderCollider, collider, ignore);
```

Comment:
From what I can tell this is supposed to make it so held items don't collide with the holder, but do collide with other colliders in the scene. Right now though, held items do not react to collisions with other colliders in the scene. I think this is because the item's rigidbody is set to be kinematic on line 43. Swapping  itemToGrab.Rigidbody.isKinematic = true; with  itemToGrab.Rigidbody.useGravity = false; shoould do the trick I think. (Don't forget to reinstate gravity use when dropping an item)